vnext/local-cli/runWindows/utils/WindowsStoreAppUtils.ps1

```diff
@@ -212,7 +212,10 @@ function Start-Locally {
 
     add-type -TypeDefinition $code
     $appActivator = new-object StoreAppRunner.ApplicationActivationManager
-    $args = [system.String]::Join(" ", $argv)
+    if ($argv.Count -gt 0) {
```

Comment:
FYI @asklar the function parameter was already optional ```[Parameter(Mandatory=$false, Position=1,```, so I'm just adding a check to prevent the join() function failing, as it did in the CI loop.